README.rst

```diff
@@ -326,15 +326,25 @@ configuration from this path unless explicitly requested by setting
  # user variables are synchronised.
  metadata_set = nid , type, sdc_aware, nid_version
 
- # if true, ignore public images of other tenants. That is, an image is upload
- # even when a image with the same name and content exist in the regional
- # server, if this image is not owned by the tenant specified in the credential.
- # Usually, it is convenience to set this value to True. It some images were
- # owned by other tenants, a better option is to change their owner.
+# When the software asks for the list of images in a region, it gets both the
+# images owned by the tenant and the public images owned by other tenants.
+# If this parameter is true (the default and recommended value), only the
+# images of the tenant are considered.  This implies than after a synchronisation
+# can be a new image with the same name than a public image from other user,
+# which may be confusing (indeed, a warning is printed when detected), but usually
+# it is not convenience to work with images of other tenants. A better
+# alternative than putting this parameter to False is to change the owner of
```

Comment:
The previous sentences was talking about the True value of the parameter, now you mention that the better alternative to configure False...